client/src/config.js

```diff
@@ -2,7 +2,8 @@ export const api = process.env.REACT_APP_API;
 
 export const imgServer = process.env.REACT_APP_IMG_URL;
 
-export const amplitudeKey = process.env.REACT_APP_AMP_KEY;
+//Had to set the key manually or else it wouldn't work. 
+export const amplitudeKey = '3d3d8925ed298021fdfb0bb4397aa8d8';
```

Comment:
Keep the `process.env.REACT_APP_AMP_KEY;`, this exposes our Amplitude API key to others looking at the repository